doc/source/index.rst

```diff
@@ -261,6 +261,7 @@ Getting Involved
    tune-schedulers.rst
    tune-searchalg.rst
    tune-package-ref.rst
+   tune-advance-tutorial.rst
```

Comment:
let's put this right after the walkthrough

Reply:
sure. Also fix the typo --> 'advanced'